layers/+lang/clojure/funcs.el

```diff
@@ -224,7 +242,7 @@ in your Spacemacs configuration:
              clojure-enable-clj-refactor t)
     ) ")))
 
-(defmacro spacemacs|forall-clojure-modes (m &rest body)
+(defmacro spacemacs||forall-clojure-modes (m &rest body)
```

Comment:
Maybe we can keep it public as users might find this macro handy for their custom config.

Reply:
Yep, btw we should add a line to the conventions defining that a macro with `||` is private. Right now we only have this for functions and variables.